navbar.html

```diff
@@ -42,7 +42,11 @@
         </div>
         <div br-slot="br-navbar-toolbar">
           <div class="navbar-right">
-            <div br-slot="br-navbar-tool"></div>
+            <div class="btn-toolbar">
```

Comment:
It's more about the semantics of it. Bootstrap probably expects only buttons to be in a btn-toolbar. That tells me that we should have two slots here ... one for entirely custom navbar-tools (br-navbar-tool) and then a built-in, by default empty, button toolbar that has a "br-navbar-toolbar-button" slot.